pkg/operation/seed/seed.go

```diff
@@ -54,6 +55,11 @@ import (
 
 const (
 	caSeed = "ca-seed"
+
+	// fluentBitConfigMapKubernetesFilterConfig is a constant for the Fluent Bit ConfigMap's section regarding Kubernetes filters
+	fluentBitConfigMapKubernetesFilter = "filter-kubernetes.conf"
```

Comment:
I think these consts should be exposed in `pkg/apis/core/v1alpha1` so that extensions can easily use them.